src/style.css

```diff
@@ -2,0 +1,60 @@
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir.ttc') format('truetype');
+  font-weight: 300;
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.search-bar__form {
+  margin: 0;
+  padding: 0;
+  width: 100%;
+}
+
+.search-bar__item::placeholder {
+  color: rgba(61, 78, 97, 1);
+}
+
+.search-bar__item {
+  display: block;
+
+  margin-top: 20px;
+  width: 100%;
+  height: 42px;
+  font-family: Avenir, sans-serif;
```

Comment:
It's a good practice to provide fallback fonts for better compatibility across different systems. Consider adding a generic font family as a fallback, e.g., `font-family: Avenir, sans-serif;`.